src/javascripts/cookies.js

```diff
@@ -0,0 +1,43 @@
+// Add cookie to session
+
+(function() {
+  var
+    cookie = false,
+    cookieContent = document.getElementById('cookie');
+
+  checkCookie();
+  if (cookie === true) {
+    cookieContent.style.display = 'none';
+  } else {
+    cookieContent.style.display = 'block';
+  }
+
+  function setCookie(cname, cvalue, exdays) {
```

Comment:
`setCookie` could be a global function for reuse elsewhere

Reply:
It may also be worth checking if a cookie with that particular name already exists